### ISSUE TITLE
voice channels like mumble, discord, or other game-

### ISSUE THREAD

**Issue by sinteur** [RocketChat/Rocket.Chat#3661]
For game support there exists services that allow groups of people to share a voice chat. A good open source example is mumble: http://wiki.mumble.info/wiki/Main_Page

It would be nice to have this available for all people in a channel

**Comment by Disar** [comment]
Yes this would be nice. I use Rocket.chat with my development team but some of our members are not located here at where we work. It would be nice to create a voice channel within the same room so I can directly talk to them without having to create a new room or make everyone else have it blasted trough their headphones.

**Comment by LeDistordu** [comment]
+1

currently my team use Rocket.Chat + Mumble !

Just find an idea to be notified when users use vocal. I think speaker icon on right side with number of voices detected is a good idea.

When you click on this icon you can set parameter
- Unmute / Mute
- Voice / Voiceless
- Record

+ bbcode for notify user to use vocal :)

**Comment by JohnnyKing94** [comment]
+1 I support the idea... it would be nice and a good competitor against Discord atm

**Comment by royalaid** [comment]
+1 I would also love to link my mumble server up to RC and have nice unified access

**Comment by cryptikon** [comment]
One of the few things discord still does better than Rocket.Chat. I would really appreciate this.

**Comment by Noculi** [comment]
While this is a cool idea, it would not fit the design of RC.
A substitute can be a "drop-in" type setting that looks something like the screenshot below.


![notincall](https://user-images.githubusercontent.com/15057172/28344295-5821f99c-6bd7-11e7-85ad-724f5a785f6c.png)
![incall](https://user-images.githubusercontent.com/15057172/28344296-582adf8a-6bd7-11e7-9473-3f15c34d02d5.png)

**Comment by wiredgorilla** [comment]
How could you incorporate Discord with RC? Is there a way to do so ?

**Comment by zzzkeil** [comment]
+1

**Comment by Per0x** [comment]
Any news about this feature ? I'm thinking of going back to Discord just because of this lack.

**Comment by STX2k** [comment]
I don't understand why this feature will not added.. it is an killer feature! 

+1

**Comment by nixpc** [comment]
@marceloschmidt What is the reason behind not planning to add this? It seems like it would be an amazing feature for RocketChat to have.

**Comment by marceloschmidt** [comment]
Our statement is that we will not be building this ourselves, but like stated before, we're open to pull requests with that feature.

**Comment by XGhozt** [comment]
+1

**Comment by poVoq** [comment]
https://github.com/Johni0702/mumble-web/tree/webrtc
Is an implementation of Mumble over webRTC. Could probably be adapted for rocket.chat.

**Comment by koszycl** [comment]
+1

**Comment by m0rtis0** [comment]
Why is it now, in the age of Covid, still not possible that even one company that makes FOSS software thinks about implementing this. If you wanted one perfect moment to steal literally 50% of Discords userbase while they are being sold to Microsoft this would be it. Sorry for being the angry off-topic guy but I just cannot believe it

**Comment by joelchornik** [comment]
+1 on this. Left rocket for discord mainly because of this feature.

**Comment by Blueshawk** [comment]
> +1 on this. Left rocket for discord mainly because of this feature.

+1 here as well, trying to leave discord but need a better audio option. Jitsi is a mess.

**Comment by DillFrescott** [comment]
Amazing RC doesn't have this nor do they seem to want it judging by some of these responses from the devs. Insanity!!!!

**Comment by DillFrescott** [comment]
@marceloschmidt Sir, sorry to ping you, but just letting you know that implementing voice/video channels on rocket chat would make sooooo many more people use rocket chat. Don't get me wrong, it's a great platform the way it is, but having this feature would be an absolute goldmine for rocket chat.

**Comment by janilowski** [comment]
just letting you know that implementing voice/video channels on rocket chat would make sooooo many more people use rocket chat


I am an admin of a Discord server with more than 600 active users. It recently got taken down because Discord - no discernible reason for the deletion.
Me and my community are looking for an alternative. We just rejected Rocket.Chat specifically because of little support for voice channels.

**Comment by terR0Q** [comment]
This could be great feature for RC.
